common/transport/amqp/devdoc/receiver_link_requirements.md

```diff
@@ -42,33 +42,33 @@ receiverLink.on('message', (msg) => {
 
 ### attach(callback: (err?: Error) => void): void
 
-**SRS_NODE_AMQP_RECEIVER_LINK_16_004: [** The `attach` method shall use the stored instance of the `amqp10.AmqpClient` object to attach a new link object with the `linkAddress` and `linkOptions` provided when creating the `ReceiverLink` instance. **]**
+**SRS_NODE_AMQP_RECEIVER_LINK_16_004: [** The `attach` method shall use the stored instance of the `rhea` session object to attach a new link object with the combined `linkAddress` and `linkOptions` provided when creating the `ReceiverLink` instance. **]**
 
-**SRS_NODE_AMQP_RECEIVER_LINK_16_006: [** The `ReceiverLink` object should subscribe to the `detached` event of the newly created `amqp10` link object. **]**
+**SRS_NODE_AMQP_RECEIVER_LINK_16_006: [** The `ReceiverLink` object should subscribe to the `receiver_close` event of the newly created `rhea` link object. **]**
 
-**SRS_NODE_AMQP_RECEIVER_LINK_16_007: [** The `ReceiverLink` object should subscribe to the `errorReceived` event of the newly created `amqp10` link object. **]**
+**SRS_NODE_AMQP_RECEIVER_LINK_16_007: [** The `ReceiverLink` object should subscribe to the `error` event of the newly created `rhea` link object. **]**
 
-**SRS_NODE_AMQP_RECEIVER_LINK_16_008: [** If the `amqp10.AmqpClient` fails to create the link the `callback` function shall be called with this error object. **]**
+**SRS_NODE_AMQP_RECEIVER_LINK_16_008: [** If the `rhea` session fails to create the link the `callback` function shall be called with this error object. **]**
```

Comment:
>If the `rhea` session fails to create [](start = 42, length = 37)

how is that failure materialized? an event?

Reply:
Yes.

---
In reply to: [191908138](https://github.com/Azure/azure-iot-sdk-node/pull/300#discussion_r191908138) [](ancestors = 191908138)